postgres/src/app.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from fastapi import Depends, FastAPI, HTTPException, status
+from sqlalchemy import text
+from sqlalchemy.exc import IntegrityError
+
+from src.db import get_db
+from src.model import Task, Tasks
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+app = FastAPI()
+
+
+@app.post("/tasks", status_code=status.HTTP_201_CREATED)
```

Comment:
```suggestion
@app.post("/task", status_code=status.HTTP_201_CREATED)
```
At the moment, the app allows for sending only one task